index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="container-top">
+            <h1>Rita Daniel</h1>
+            <img src="./images/Union2x.svg" alt="menu-bar">
+        </div>
+        <div class="primary-text">
+            <h2>Hey There. I’m Rita</h2>
+            <h3>I’am a Software Developer</h3>
```

Comment:
- [ ] Kindly I recommend you to use `h1` tag for main title and `h2` tag for sub-title , This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.